docs/dev_install.rst

```diff
@@ -116,6 +116,8 @@ Setting up Taichi for development
 
     Then execute ``source ~/.bashrc`` to reload shell config.
 
+    If you're using fish, use ``set -x NAME VALUES``, otherwise it won't be loaded by child processes.
```

Comment:
Sounds great! I only have 1 nit:
```suggestion
    .. note::

        If you're using fish, use ``set -x NAME VALUES``, otherwise it won't be loaded by child processes.
```

Reply:
Great! I'd adopt this.